src/sqlfluff/dialects/dialect_oracle.py

```diff
@@ -616,3 +616,19 @@ class SqlplusVariableGrammar(BaseSegment):
             Ref("ParameterNameSegment"),
         )
     )
+
+
+class FetchClauseSegment(BaseSegment):
+    """A `FETCH` clause like in `SELECT."""
+
+    type = "fetch_clause"
+    match_grammar: Matchable = Sequence(
+        "FETCH",
+        OneOf(
+            "FIRST",
+            "NEXT",
+        ),
+        Ref("NumericLiteralSegment", optional=True),
```

Comment:
Hmm, I think better to just adding optional like `Ref("NumericLiteralSegment", optional=True),` to ansi? I think better to loosen up ansi instead of copying most of the function again here.

Reply:
Ok, and the test files I put in ansi folder ?

Reply:
I’d say fine to leave tests where they are.